src/components/Training/Modal.js

```diff
@@ -24,7 +24,7 @@ const Wrapper = styled.div`
   opacity: 0;
   top: 0;
   left: 0;
-  z-index: 9999;
+  z-index: ${props => props.theme.zIndexes.modal};
```

Comment:
When you start caring about z indexes you are a senior

Reply:
hahah, I've had my share with z-index problems so now I like to have the main ones in a central place.
Also, it's useful when you want to position things relative to each other like:

```
  z-index: ${props => props.theme.zIndexes.modal + 1};

```

Reply:
See? Senior